benchmarks/import_xdsl_opt.py

```diff
@@ -0,0 +1,12 @@
+# Write the benchmarking functions here.
```

Comment:
Because asv uses weird magic to run these files, I don't think they run them as normal Python, so I thought it made sense to separate the two folders. Numpy and scipy have python scripts to run asv, and they all do it with subprocess.call

Reply:
Can we then change the name to reflect that?
Could we rename `benchmarks` to `asv_bench` and put it inside the `bench` folder?

Reply:
Sure